examples/queries.md

```
@@ -452,6 +452,32 @@ args: ["2024-01-01" "2024-02-01" "archived" "zzz"]
 - `Between` e `NotBetween` aceitam qualquer expressão, inclusive `Raw` ou subconsultas.
 - Vários predicados informados em `Where` continuam agrupados por `AND`.
 
+### 13.1 Filtro com NOT IN usando CastAsAny
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)